back/admin/settings/email_template_views.py

```
@@ -0,0 +1,158 @@
+import json
+from django.contrib.messages.views import SuccessMessageMixin
+from django.http import JsonResponse
+from django.urls import reverse_lazy
+from django.utils.translation import gettext_lazy as _
+from django.views.decorators.http import require_POST
+from django.views.generic import CreateView, DeleteView, ListView, UpdateView
+from django.contrib.auth.decorators import login_required
+
+from users.mixins import AdminPermMixin, LoginRequiredMixin
+from users.models import User
```

Ruff (0.11.9)

`users.models.User` imported but unused

Remove unused import: `users.models.User`

(F401)